src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.people')}
+                    onCloseButtonPress={() => Navigation.dismissModal(true)}
+                />
+                <ConfirmModal
+                    danger
+                    title={this.props.translate('workspace.people.removeMembersTitle')}
+                    isVisible={this.state.isRemoveMembersConfirmModalVisible}
+                    onConfirm={this.removeUsers}
+                    onCancel={this.hideConfirmModal}
+                    prompt={this.props.translate('workspace.people.removeMembersPrompt')}
+                    confirmText={this.props.translate('common.remove')}
+                    cancelText={this.props.translate('common.cancel')}
+                />
+                <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                    <View style={styles.pageWrapper}>
+                        <Text style={[styles.mb6, styles.textP, styles.textFull]}>
+                            {this.props.translate('workspace.people.tagline')}
+                        </Text>
+                        <View style={styles.buttonRow}>
+                            <Button
+                                success
+                                style={[]}
+                                text={this.props.translate('common.invite')}
+                                onPress={this.inviteUser}
+                            />
+                            <Button
+                                danger
+                                style={[styles.ml2]}
+                                isDisabled={this.state.selectedEmployees.length === 0}
+                                text={this.props.translate('common.remove')}
+                                onPress={this.askForConfirmationToRemove}
+                            />
+                        </View>
+                        <View style={[styles.w100, styles.mt4]}>
+                            {
+                                this.props.policy.employeeList ? (
+                                    <FlatList
+                                        ListHeaderComponent={this.renderHeader()}
+                                        ref={flatListRef}
+                                        renderItem={this.renderItem}
+                                        contentContainerStyle={[styles.w100]}
+                                        initialRowHeight={32}
+                                        // eslint-disable-next-line max-len
+                                        data={(this.props.policy.employeeList && this.props.policy.employeeList.length !== 0)
+                                            // eslint-disable-next-line max-len
+                                            ? this.props.policy.employeeList.map(email => this.props.personalDetails[email] || false)
+                                            : []}
```

Comment:
Rather than check the `employeeList` here it might be better to just initialize this in the beginning of `render()` with a new variable `data` in the correct shape the list wants.

Reply:
That will be better for readability, good point!